packages/web/integrations/axelar/transfer.tsx

```diff
@@ -226,7 +226,7 @@ const AxelarTransfer: FunctionComponent<
     };
 
     /** Osmosis chain ID accepted by Axelar APIs. */
-    const osmosisAxelarChainId = isTestNet ? "osmosis-5" : "osmosis";
+    const osmosisAxelarChainId = isTestNet ? "osmosis-6" : "osmosis";
```

Comment:
So osmo test 5's chain ID is `osmosis-6`?

Reply:
Only according to Axelar's API, yes
<img width="226" alt="image" src="https://github.com/osmosis-labs/osmosis-frontend/assets/95667791/a97d15a5-b13f-4d95-aca9-6d3023bd9806">

Reply:
Oh this is Axelar. My bad. Thanks, wasn't paying complete attention.